cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientRequestFilter.java

```diff
@@ -0,0 +1,57 @@
+package org.cloudfoundry.client.lib.rest;
+
+import java.net.Socket;
+import java.net.SocketException;
+
+import org.cloudfoundry.client.lib.CloudCredentials;
+import org.cloudfoundry.client.lib.oauth2.OAuthClient;
+import org.springframework.web.reactive.function.client.ClientRequest;
+import org.springframework.web.reactive.function.client.ClientResponse;
+import org.springframework.web.reactive.function.client.ExchangeFilterFunction;
+import org.springframework.web.reactive.function.client.ExchangeFunction;
+
+import reactor.core.publisher.Mono;
+
+public class CloudControllerRestClientRequestFilter implements ExchangeFilterFunction {
+
+    private static final String AUTHORIZATION_HEADER_KEY = "Authorization";
```

Comment:
Can we use some standart constant for Auhtorization header?
Now this is defined at least on two places in this project.

Reply:
e.g.
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/HttpHeaders.html#AUTHORIZATION